jbmc/regression/jbmc-strings/StartsWithConstantEvalution/constant_equals.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+RefineStrings.class
+--function string_refinement.RefineStrings.constantComparison --show-vcc --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar`
```

Comment:
Maybe I misunderstood the conversation, but it seemed to me that you wouldn't even need the library?

Reply:
(While it's not hugely important - this was needed in the end, without it String.equals is stubbed so doesn't apply the function application)